api/v1/db/oracledb/game-dao.js

```diff
@@ -88,4 +88,28 @@ const getGameById = async (id) => {
   }
 };
 
-module.exports = { getGames, getGameById, getGamesByMemberId };
+/**
+ * @summary Check the if the game with the id exists.
+ * @function
+ * @param {Int} id The id of the game
+ * @returns {Bool} If the game exists or not.
+ */
```

Comment:
I think the types should be changed a little `Int` -> `number` and `Bool` -> `boolean`. Here is some documentation on supported JSDoc types: https://jsdoc.app/tags-type.html